db/people.yml

```diff
@@ -880,6 +1017,23 @@ ekjaer:
       status: visitor-unsupported
   name: Emil Kjaer
   position: Visiting Researcher
+fatassi:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 0
+      institution: fix.me
+      degree: fix.me
+  employment:
+    - begin_year: 2005
+      begin_month: 6
```

Comment:
this is farad atassi.  He was a student at Purdue, I think in the group of Steve Byrne

Reply:
Ok, I am trying to find his begin year